src/rollback_to_stable/rts.c

```diff
@@ -143,9 +143,14 @@ __wt_rts_btree_apply_all(WT_SESSION_IMPL *session, wt_timestamp_t rollback_times
     }
     WT_ERR_NOTFOUND_OK(ret, false);
 
-    if (F_ISSET(S2C(session), WT_CONN_RECOVERING))
+    if (F_ISSET(S2C(session), WT_CONN_RECOVERING)) {
+        __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
+          WT_RTS_VERB_TAG_HS_TREE_ROLLBACK
```

Comment:
This tag seems to be reused from another message, and there's a couple below that are definitely used multiple times. They should be unique (unless the message really is identical), so that the verifier is able to tell what it should try to parse out from the message.